Sources/JavaScriptKit/JSValueConvertible.swift

```diff
@@ -194,6 +266,6 @@ extension Array where Element == JSValueConvertible {
 
 extension Array where Element: JSValueConvertible {
     func withRawJSValues<T>(_ body: ([RawJSValue]) -> T) -> T {
-        Swift.Array<JSValueConvertible>.withRawJSValues(self)(body)
+        Array<JSValueConvertible>.withRawJSValues(self)(body)
```

Comment:
This would also work if you like conciseness 🙂

```suggestion
        [JSValueConvertible].withRawJSValues(self)(body)
```